parachain/test-parachains/adder/collator/src/lib.rs

```diff
@@ -33,6 +33,25 @@ use test_parachain_adder::{execute, hash_state, BlockData, HeadData};
 /// This is a constant to make tests easily reproducible.
 const ADD: u64 = 2;
 
+/// Calculates the head and state for the given `number`.
```

Comment:
```suggestion
/// Calculates the head and state for the block with the given `number`.
```

To emphasize that it's the block number, not the number found in the state.